From 6fb003a7a117f97a35b078ba726c84adeae29c4c Mon Sep 17 00:00:00 2001
From: Ansuel Smith <ansuelsmth@gmail.com>
Date: Wed, 10 Feb 2021 10:54:19 +0100
Subject: [PATCH 2/2] Documentation: devicetree: mtd: qcom_nandc: document
 qcom,boot_layout_size binding

Document new qcom,boot_layout_size binding used to apply special
read/write confituation to boots partitions.

Signed-off-by: Ansuel Smith <ansuelsmth@gmail.com>
---
 Documentation/devicetree/bindings/mtd/qcom,nandc.yaml | 11 +++++++++++
 1 file changed, 11 insertions(+)

--- a/Documentation/devicetree/bindings/mtd/qcom,nandc.yaml
+++ b/Documentation/devicetree/bindings/mtd/qcom,nandc.yaml
@@ -78,6 +78,14 @@ allOf:
             Must contain the ADM data type CRCI block instance number
             specified for the NAND controller on the given platform
 
+        qcom,boot_pages_size:
+          description:
+            Should contain the size of the total boot partitions
+            where the boot layout read/write specific configuration
+            should be used. The boot layout is considered from the
+            start of the nand to the value set in this binding.
+            Only used in combination with 'nand-is-boot-medium'.
+
   - if:
       properties:
         compatible:
@@ -135,6 +143,9 @@ examples:
         nand-ecc-strength = <4>;
         nand-bus-width = <8>;
 
+        nand-is-boot-medium;
+        qcom,boot_pages_size: <0x58a0000>;
+
         partitions {
           compatible = "fixed-partitions";
           #address-cells = <1>;
